pkgs/development/python-modules/sphinxcontrib-tikz/default.nix

```diff
@@ -0,0 +1,30 @@
+{ lib
+, buildPythonPackage
+, fetchPypi
+, sphinx
+, pdf2svg
+, texLive
+}:
+
+buildPythonPackage rec {
+  pname = "sphinxcontrib-tikz";
+  version = "0.4.6";
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "4f362b11e3c2bd17d5f0f07fec03917c16fc5bbcda6fe31ee137c547ed6b03a3";
+  };
+
+  propagatedBuildInputs = [ sphinx pdf2svg texLive ];
```

Comment:
TeXLive should not be propagated.

Reply:
...as well as pdf2svg

Reply:
This package will not work without pdf2svg and texlive. Curious how in general this is handled?

Reply:
Both of these aren't Python packages and should hence not be propagated. Instead, their path should be patched into the source using `substituteAll`.

Reply:
Is there an example of that? I've never seen it used. Like so? https://github.com/NixOS/nixpkgs/blob/1eca8366e8d0728636d36c6da15d706acdc50bd3/pkgs/development/python-modules/pygraphviz/default.nix#L17

Reply:
Exactly :+1: